plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -79,7 +81,7 @@ export function useProductHelper() {
 	 */
 	const createProductWithStatus = useCallback(
 		async (
-			product: Omit< Product, ReadOnlyProperties >,
+			product: Omit< Partial< Product >, ReadOnlyProperties >,
```

Comment:
It might be the same end result, but it feels to me that `Partial < Omit < Product, ReadOnlyProperties > >` is a bit more clear.